eth/filters/api.go

```diff
@@ -154,15 +154,18 @@ func (api *PublicFilterAPI) NewPendingTransactions(ctx context.Context, fullTx *
 	go func() {
 		txs := make(chan []*types.Transaction, 128)
 		pendingTxSub := api.events.SubscribePendingTxs(txs)
+		chainConfig := api.backend.ChainConfig()
 
 		for {
 			select {
 			case txs := <-txs:
 				// To keep the original behaviour, send a single tx hash in one notification.
 				// TODO(rjl493456442) Send a batch of tx hashes in one notification
+				latest := api.backend.CurrentHeader()
 				for _, tx := range txs {
 					if fullTx != nil && *fullTx {
-						notifier.Notify(rpcSub.ID, tx)
+						rpcTx := ethapi.NewRPCPendingTransaction(tx, latest, chainConfig, nil)
```

Comment:
Could you add a comment here mentioning that a `basefeefn` is not required here, because it's a pending transaction?
See https://github.com/celo-org/celo-blockchain/blob/f0adb14e48288ff6d6e4d9320d5980afd4d17c2d/internal/ethapi/api.go#L1340-L1354